examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -108,7 +108,8 @@ class ReportCommand : public ModelCommand, public chip::app::ReadClient::Callbac
 protected:
     CHIP_ERROR ReportAttribute(ChipDevice * device, chip::EndpointId endpointId, chip::ClusterId clusterId,
                                chip::AttributeId attributeId, chip::app::ReadClient::InteractionType interactionType,
-                               uint16_t minInterval = 0, uint16_t maxInterval = 0)
+                               chip::DataVersion dataVersion = chip::kUndefinedDataVersion, uint16_t minInterval = 0,
```

Comment:
I just checked the spec, and there is no provision there for 0 being an invalid data version.  So kUndefinedDataVersion needs to go away, period.  We should not be adding new uses in this PR.

Please use `const Optional<DataVersion> &` to represent maybe-absent data version arguments.

Reply:
agree, done, thanks